### DISCUSSION TITLE
rename to something less copyright-y?

### DISCUSSION THREAD

**Discussion by ghost** [blackjack4494/youtube-dlc#159] [General] [answered]
im sure it's fine as long as no monetary operations are involved,
but having a potential of DMCA takedown from youtube/google/alphabet lawyers
b/c of the name alone does not seem like its worth the ever-so-minuscule but still risk

something like `YDL[C]` or whatever else, might be completely unrelated idk?

**Comment by blackjack4494** [comment, marked as answer]
There have been some talkings about this somewhere else. Some alternatives were video-dl or mget (little pun related to wget) where m would stand for media.  
Long term this project would get its own repo (no fork). But for now it's easier to manage that way also the naming problem isn't solved yet. Furthermore I want to implement some more changes like trying to implement some plugin system.

**Reply by ghost** [reply]
mget might be interpreted as intentionally typo'ed "miget", so rather not that one

**Reply by blackjack4494** [reply]
Miget? Do you mean midget? Or what does miget stand for?  
Media-dl (mdl) would work too I guess

**Reply by ghost** [reply]
hm, yeah, somehow i read it like "gadet"

**Reply by SeppSpenderRind** [reply]
Just an idea I had: `hen-dl` could be a suitable name!
In Ancient Greek, "hen" means "The One", so "hen-dl" would be _the one_ tool to download media with.
(on top of that, "Hendl" means "(roasted) chicken" in Bavarian and Austrian German, so there would be something like an iconic mascot, too) :D
